compiler/luci/service/src/Nodes/CircleStridedSlice.cpp

```diff
@@ -33,4 +35,14 @@ luci::CircleNode *CloneNodeLet<CN::STUV>::visit(const luci::CircleStridedSlice *
   return cloned;
 }
 
+namespace sinf
+{
+
+loco::TensorShape Algorithm::visit(const luci::CircleStridedSlice *node)
+{
+  return infer_output_shape(node);
```

Comment:
Please move the codes of `infer_output_shape` in this file :)
i.e.,

```suggestion
  // implementation codes of infer_output_shape function
```

Reply:
Thank you for the feedback! I'll move the `infer_output_shape` codes and remove the mentioned files.